src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1130,6 +1170,8 @@ def load_arguments(self, _):
         c.argument('service_endpoints', nargs='+', min_api='2017-06-01')
         c.argument('service_endpoint_policy', nargs='+', min_api='2018-07-01', help='Space-separated list of names or IDs of service endpoint policies to apply.', validator=validate_service_endpoint_policy)
         c.argument('delegations', nargs='+', min_api='2017-08-01', help='Space-separated list of services to whom the subnet should be delegated. (e.g. Microsoft.Sql/servers)', validator=validate_delegations)
+        c.argument('private_endpoint_network_policies', arg_type=get_enum_type(['Enabled', 'Disabled']), min_api='2019-04-01', help='Enable or Disable private endpoint on the subnet.')
```

Comment:
wouldn't it be better to have **--enable-private-endpoint-network-policies** and **--enable-private-link-service-network-policies** without enum?

Reply:
good idea. I will solve this.